packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/UIDEditor/index.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActionButton,
+  DialogActions,
+  DialogCancelButton,
+  DialogContent,
+  DialogHeader,
+  FormInput,
+  Icon,
+  Text,
+} from "@prismicio/editor-ui";
+import {
+  Group,
+  NestableWidget,
+  UID,
+} from "@prismicio/types-internal/lib/customtypes";
+import { Formik } from "formik";
```

Comment:
❓ Using formik for this little form seems overkill. I don't know if we want to sue formik for everything, or just for complex forms, or not use it at all because it might be a legacy thing. Worth asking @xrutayisire.